crates/utils/src/launch.rs

```diff
@@ -0,0 +1,51 @@
+use std::{
+    collections::HashMap,
+    io::{BufRead, BufReader, Write},
+    net::TcpStream,
+};
+
+use anyhow::{anyhow, Result};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Launch {
+    pub binary: String,
+    pub args: Vec<String>,
+    pub envs: HashMap<String, String>,
+}
+
+pub fn request_launch(
```

Comment:
I think this should live in the `krun` crate?

Reply:
Hm... okay.

Reply:
The way I see it, the `utils` crate should be for stuff that are common between the different crates / binaries. And any dependency added to the `utils` crate should ideally be optional, gated behind a feature flag.

Reply:
Done.